pytorch_lightning/plugins/training_type/parallel.py

```diff
@@ -83,6 +84,14 @@ def reduce_early_stopping_decision(self, should_stop: bool) -> bool:
         should_stop = bool(should_stop == self.world_size)
         return should_stop
 
+    @property
+    def torch_distributed_backend(self):
+        torch_backend = os.getenv("TORCH_DISTRIBUTED_BACKEND")
+        if torch_backend is None:
+            torch_backend = "nccl" if self.on_gpu else "gloo"      
+        rank_zero_info(f"torch.distributed backend is {torch_backend}.")
```

Comment:
Unsure about this log. 99.9% of people won't care about it

Reply:
Removed !